portal/src/components/Materials/MaterialSet/MaterialPartOfSet.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <h3 class="material__info_subttl">{{ $t('Part-of-set') }}</h3>
+    <div class="material__info_part_of">
+      <div class="material__info_main" @click="goToMaterial">
+        <h3>
+          {{ mainMaterial.title }}
+          <i class="material__info_main_arrow"></i>
+        </h3>
+        <div v-if="mainMaterial.has_part" class="">
+          {{ $tc('Materials', mainMaterial.has_part.length + 1) }}
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'MaterialPartOfSet',
+  props: {
+    material: {
```

Comment:
kan je die warnings nog fixen die hieronder staan?

Reply:
dat heb ik tot in den treure geprobeerd, maar dat lukt me niet

Reply:
ook niet als default null is? Ik zal zo ook wel even proberen